array_1.py

```diff
@@ -1,333 +1,339 @@
 for metadata in class_resources:
     # Code that does parsing stuff
-    page = requests.get(metadata['url'], cookies=cookies)
+    page = requests.get(metadata["url"], cookies=cookies)
     print(page.text)
 
-    soup = BeautifulSoup(page.text, 'html.parser')
+    soup = BeautifulSoup(page.text, "html.parser")
 
     # Initialize an empty list to store URLs
     list_of_urls_on_page = []
+    # list_of_urls_on_page_a = []
+    #list_of_urls_on_page_iframe = []
 
-    if metadata['title'] == "Computer Science" or metadata['title'] == "Calculus":
+    if metadata["title"] == "Computer Science" or metadata["title"] == "Calculus":
         # Enters a loop to iterate over all <iframe> tags on the webpage using BeautifulSoup's find_all method
         # these specific iframes are an embeded youtube clients
-        for iframe_tag in soup.find_all('iframe'):
+        for iframe_tag in soup.find_all("iframe"):
             # For each <iframe> tag, it extracts the 'src' attribute (the URL) using iframe_tag.get('src') and appends it to list_of_urls_on_page
-            list_of_urls_on_page.append([iframe_tag.get('src'), iframe_tag.text])
-    elif metadata['title'] in ["Physics First Semester", "Physics Second Semester" "Chemistry First Semester", "Chemistry Second Semester", "AP Chemistry"]: 
-        for iframe_tag in soup.find_all('iframe'):
+            list_of_urls_on_page.append([iframe_tag.get("src"), iframe_tag.text])
+    elif metadata["title"] in [
+        # all of these -- can get <a> , can't get <iframe> DATA
+        "Physics First Semester",  # Links are wrong -- Gave range, but is wrong index
+        "Physics Second Semester",  # Links are wrong -- Gave range, but is wrong index
+        "Chemistry First Semester",
+        "Chemistry Second Semester",
+        "AP Chemistry",
+    ]:
+        for iframe_tag in soup.find_all("iframe"):
             # Iterate over all <iframe> tags on the webpage and append them to empty url list
-            list_of_urls_on_page.append([iframe_tag.get('src'), iframe_tag.text])
+            list_of_urls_on_page.append([iframe_tag.get("src"), iframe_tag.text])
 
-        for a_tag in soup.find_all('a'):
+        for a_tag in soup.find_all("a"):
             # Iterate over all <a> tags on the webpage and append them to empty url list
-            list_of_urls_on_page.append([a_tag.get('href'), a_tag.text]) 
+            list_of_urls_on_page.append([a_tag.get("href"), a_tag.text])
 
-    else: 
-        for a_tag in soup.find_all('a'):
-            list_of_urls_on_page.append([a_tag.get('href'), a_tag.text])
+    else:
+        for a_tag in soup.find_all("a"):
+            list_of_urls_on_page.append([a_tag.get("href"), a_tag.text])
 
     print("---------------------------- beginning of iteration")
     print(metadata)
-    print(count, metadata['title']," - ",metadata['url'])
-    count = count+1
+    print(count, metadata["title"], " - ", metadata["url"])
+    count = count + 1
```

Comment:
Why did you add a space here around the plus sign? I don't like that formatting.

Reply:
this is not a democracy